transaction/context/tracked/tracked.go

```diff
@@ -0,0 +1,26 @@
+// Package tracked stores and accesses the information of if a transaction is
+// already tracked in and from context.Context.
+package tracked
```

Comment:
This context package manages the information about the tracking state of a transaction. In case we already tracked a response we want to know this in several places. This information is propagated within some the given request context. That way we know the tracking state of a transaction across an endpoints methods.